packages/notification-controller/README.md

```diff
@@ -1,6 +1,6 @@
 # `@metamask/notification-controller`
 
-Manages display of the "What's New" messages in MetaMask.
+Manages display of notifications within MetaMask (Flask only).
```

Comment:
Hmm. Not sure we want to say "Flask only" here. That's a reflection of how this is being used today, not really an inherent property of this controller. It will be used for stable eventually.

Reply:
Agreed, let's just stick to specifying what the controller does.

Reply:
Heh, another one 😅